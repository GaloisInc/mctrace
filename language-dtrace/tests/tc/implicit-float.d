proc:::exec_success {
  global_float = 5.48f;
}
