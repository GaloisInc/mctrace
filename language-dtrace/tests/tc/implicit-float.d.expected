TCExpected { globals = [("global_float", FloatType)]
         }
